src/pages/DetailPage/DetailInfo.js

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components';
+import Detailoption from './Detailoption';
+
+const DetailInfo = () => {
+  return (
+    <Wrapdiv>
+      <section>
+        <Flexdiv>
+          <Namespan>침대1</Namespan>
+          <div>
+            <img src="../images/ribbon.png" alt="ribbon" />
+            <img src="../images/share.png" alt="share" />
+          </div>
+        </Flexdiv>
+        <Pricediv>
+          <span>1,000</span>
+          <span>원</span>
+          <div>특가</div>
+        </Pricediv>
+        <Infodiv>
+          <Subspan>혜택</Subspan>
+          <span>78p 적립(WELCOME 0.3%적립)</span>
+        </Infodiv>
+        <Infodiv>
+          <Subspan>배송</Subspan>
+          <span>3,000원 (50,000원 이상 구매시 무료배송)</span>
+        </Infodiv>
+      </section>
+      <Detailoption />
+    </Wrapdiv>
+  );
+};
+
+const Flexdiv = styled.div`
+  display: flex;
+  div {
+    margin-left: 350px;
+  }
+`;
+const Wrapdiv = styled.div`
```

Comment:
```suggestion
const Wrap = styled.div`
```
스타일드 컴포넌트 이름에 태그 이름을 명시해줄 필요 없을 것 같습니다.

Reply:
넵!!